Sources/ScipioKit/Producer/Cache/CacheSystem.swift

```diff
@@ -96,6 +96,7 @@ extension PinsStore.PinState: @retroactive Hashable {
 }
 
 public struct SwiftPMCacheKey: CacheKey {
+    public var canonicalPackageLocation: String
```

Comment:
I'm worried about the remote URL being embedded. If it's used in a mirrored repository, all caches will be expired. I think it's better to be it `Optional` and it should have only for local packages.

Reply:
Thanks for the suggestion, I addressed this in 507be71.